models/sklearn_extra_trees.py

```diff
@@ -0,0 +1,116 @@
+import datatable as dt
+import numpy as np
+from h2oaicore.models import CustomModel
+from sklearn.ensemble import ExtraTreesClassifier, ExtraTreesRegressor
+from sklearn.preprocessing import LabelEncoder, StandardScaler
+
+
+class ExtraTreesModel(CustomModel):
+
+    _regression = True
+    _binary = True
+    _multiclass = True
+
+    _display_name = "ExtraTrees"
+    _description = "Extra Trees Model based on sklearn"
+
+    # def make_instance(self):
+    #     return self.__class__._class(seed=self.random_state)
+
+    def set_default_params(self, accuracy=None, time_tolerance=None,
+                           interpretability=None, **kwargs):
+        n_jobs = -1
+        n_estimators = min(kwargs.get('n_estimators', 10), 1000)
+        criterion = kwargs['metric'] if "metric" in kwargs and kwargs["metric"] in ["gini", "entropy"] else "gini"
+        self.params = {'n_estimators': n_estimators,
+                       'criterion': criterion,
+                       'n_jobs': n_jobs}
+
+    def mutuate_params(self, accuracy=None, time_tolerance=None, interpretability=None, **kwargs):
+        n_jobs = -1
+
+        if accuracy > 8:
+            estimators_list = [100, 200, 300, 500, 1000, 2000]
+        elif accuracy >= 5:
+            estimators_list = [50, 100, 200, 300, 400, 500]
+        else:
+            estimators_list = [10, 50, 100, 150, 200, 250, 300]
+
+        criterion = kwargs['metric'] if "metric" in kwargs and kwargs['metric'] in ["mae", "mse", "gini", "entropy"] else "gini"
```

Comment:
Can remove if this is set appropriately in `set_default_parameters`
```suggestion
        # Parameters that don't need to be tuned
        n_jobs = self.params.get('n_jobs')
        param_criterion = self.params.get('criterion')
```